custom/conf/app.example.ini

```diff
@@ -423,7 +423,15 @@ STARTUP_TIMEOUT=30s
 
 ; repo indexer by default disabled, since it uses a lot of disk space
 REPO_INDEXER_ENABLED = false
+; Code search engine type, could be `bleve` or `elasticsearch`.
+REPO_INDEXER_TYPE = bleve
+; Index file used for code search.
 REPO_INDEXER_PATH = indexers/repos.bleve
+; Code indexer connection string, available when `REPO_INDEXER_TYPE` is elasticsearch. i.e. http://elastic:changeme@localhost:9200
+REPO_INDEXER_CONN_STR = 
+; Code indexer name, available when `REPO_INDEXER_TYPE` is elasticsearch
+`REPO_INDEXER_NAME` = gitea_codes
```

Comment:
What's going on here? Why the back-ticks?

Reply:
```suggestion
REPO_INDEXER_NAME = gitea_codes
```

Reply:
O another one

Reply:
*hmm did commented on it but was resolved ?!?